ibridges/meta.py

```diff
@@ -479,13 +495,16 @@ def update(self, new_key: str, new_value: str, new_units: Optional[str] = None):
 
         """
         new_item_key = (new_key, new_value, new_units)
+        print(new_item_key)
         try:
             _new_item = self._ibridges_meta[new_item_key]
         except KeyError:
-            self._ibridges_meta.add(*new_item_key)
             try:
+                self._ibridges_meta.add(*new_item_key)
```

Comment:
I think it should be in the old place. Now, we don't know whether the new one was already added.